MANIFEST.in

```diff
@@ -43,6 +43,7 @@ include Pipfile Pipfile.lock
 include pyproject.toml
 include pytest.ini
 include renku/data/defaults.ini
+include renku/data/default_sbatch_template.sh
```

Comment:
A common convention is to call jinja2 templates <filename>.tpl, so `renku/data/default_sbatch_template.sh.tpl`